Docs/docfx.json

```diff
@@ -67,11 +67,12 @@
       "_appTitle": "Libplanet",
       "_appLogoPath": "images/logo.svg",
       "_appFaviconPath": "images/favicon.ico",
-      "_appFooter": "Copyright © 2019 <a href=\"https://planetariumhq.com/\">Planetarium</a><br>Generated by DocFX"
+      "_appFooter": "Copyright © 2019\u20132020 <a href=\"https://planetariumhq.com/\">Planetarium</a>"
```

Comment:
Is it okay that we remove this mention?

Reply:
I think so?  Should we remain that sentence?